app/Features.tsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import Image from "next/image";
+
+export default function Features() {
+  return (
+    <section>
+      {/* This is for the Features header */}
+      <div>
+        <div className="flex flex-col items-center justify-center mt-15 lg:mt-16">
+          <div>
+            <p className="w-44 font-sans text-xl text-center text-black tracking-widest sm:text-4xl lg:text-5xl">Features</p>
```

Comment:
can you remove `text-black` here? if yes remove it so that we can simplify the code